src/Pages/SignIn/SignIn.css

```diff
@@ -0,0 +1,102 @@
+
+.login-container{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
+    margin-top: 1.5rem;
+    background: var(--gray-bg-color);
+}
+
+.login-heading{
+    margin-top: 3rem;
+    font-size: 1.8rem;
+}
+
+.login-card{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
+    padding: 1rem 3rem 2rem 3rem;
+    margin: 3rem 0 3rem 0;
+    border-radius: 10px;
+    box-shadow: 1px 1px 10px 10px rgba(0,0,0,0.1);
```

Comment:
changes: use css variable for color here because you are using in multiple places